vassal-app/src/main/java/VASSAL/counters/Deck.java

```diff
@@ -1708,6 +1708,9 @@ public void promptForDragCount() {
     }
   }
 
+  private JButton okButton;
+  private JButton cancelButton;
```

Comment:
These could be local variables in the function following?

Reply:
Also done! Also whee!